quickcommit.sh

```diff
@@ -0,0 +1,3 @@
+git add .
+git commit -m $1
+git push origin jPlateComponents
```

Comment:
Let's remove this file as well. This should be an alias/utility available on your machine rather than on the GitHub repo.